conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -55,6 +54,7 @@ type algodImporter struct {
 	ctx     context.Context
 	cancel  context.CancelFunc
 	mode    int
+	genesis *sdk.Genesis
```

Comment:
Keep the config in the `algodImporter` struct to avoid having to call the genesis endpoint after `Init()`